en/components/drop_down.md

```diff
@@ -224,6 +224,38 @@ If the sample is configured properly, a list  of countries should be displayed a
     </button>
 </div>
 
+### igxDropDownItemNavigation directive
+
+To enable keyboard navigation for the igxDropDown component, the igxDropDownItemNavigation directive should be applied. It's recommended that the directive is applied to the element that is triggering the opening and closing the of the drop down. That element will also have the focus, and when using the directive it will handle all triggered events. By default the focus will always stay on the element that igxDropDownItemNavigation is applied on, and this happens because the igxDropDown doesn't focus its items by default.
```

Comment:
"To enable keyboard navigation for the igxDropDown component, the igxDropDownItemNavigation directive ~should~**can** be applied. ~It's recommended that t~**T**he directive ~is~**should be** applied to the **active (focused)** element **or a parent container** ~is triggering the opening and closing the of the drop down~. ~That element will also have the focus, and when using the directive it will~**This will allow the directive to** handle all triggered events. By default ~the focus will always stay on the element that igxDropDownItemNavigation is applied on, and this happens because~ the igxDropDown doesn't **take** focus ~its items by default.~ **so for example the directive can be placed on a button or input that controls the drop down. When the drop down's `allowItemsFocus` is enabled the directive can be placed either on the component itself or on each item.**"

Reply:
TBH, this bit should also list the actual keyboard navigation the directive provides or link to one of the specs that describe it.